libvast/test/system/active_partition.cpp

```diff
@@ -111,18 +126,18 @@ TEST(No dense indexes serialization when create dense index in config is false)
     last_written_chunks;
   auto filesystem = sys.spawn(dummy_filesystem, std::ref(last_written_chunks));
   const auto partition_id = vast::uuid::random();
-  const std::string input_store_id = "some-id";
-  auto header_data = 3523532ull;
-  auto partition_header
-    = vast::chunk::make(&header_data, sizeof(header_data), []() noexcept {});
-  auto sut
-    = sys.spawn(vast::system::active_partition, partition_id,
-                vast::system::accountant_actor{}, filesystem, caf::settings{},
-                index_config_, vast::system::store_actor{}, input_store_id,
-                partition_header, std::make_shared<vast::taxonomies>());
+  // FIXME: We should implement a mock store and use that for this test.
```

Comment:
Wouldn't it already suffice to use the default store with the memory filesystem here?

Regardless, do you still want to fix this within this PR? Otherwise `s/FIXME/TODO`.